examples/nft-nd-nns/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22
 
 require (
 	github.com/nspcc-dev/neo-go v0.106.4-0.20241016130346-d8e945978af6
-	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20241212130705-ea0a6114d2d6
+	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20241219193620-20b74b038f23
```

Comment:
neo-go dependency should be updated afterwords in a separate commit.

Reply:
similar to https://github.com/nspcc-dev/neo-go/commit/c6d259b28577725a971987157c0e0be37dd763da ? My `./scripts/update_deps.sh 69c75acb126038a719d8a6ed3097fb20a11d50d1` update all of them and if update only internal dependencies then Tests / Check internal dependencies (pull_request) fails.

Reply:
> similar to https://github.com/nspcc-dev/neo-go/commit/c6d259b28577725a971987157c0e0be37dd763da ?

No. This commit updates `interop` dependencies. What you need is for `nft-nd-nns` manually run `go get github.com/nspcc-dev/neo-go@...` with latest commit in your PR.

Reply:
its not okay that the test is still failing?

Reply:
No. You firstly need to execute `scripts/update-deps.sh`, then commit changes, then push them into master, then manually upgrade `neo-go` dependency for NNS.